app/modules/topics/actionTypes.js

```diff
@@ -25,18 +29,24 @@ export const EDIT: 'topics/EDIT' = 'topics/EDIT';
 export const REMOVE: 'topics/REMOVE' = 'topics/REMOVE';
 export const GET: 'topics/GET' = 'topics/GET';
 export const GET_ALL_BY_USERID: 'topics/GET_ALL_BY_USERID' = 'topics/GET_ALL_BY_USERID';
+export const SAVE_CONTENT: 'topics/SAVE_CONTENT' = 'topics/SAVE_CONTENT';
+export const LOAD_CONTENT: 'topics/LOAD_CONTENT' = 'topics/LOAD_CONTENT';
```

Comment:
Either remove the `_CONTENT` suffix here or add it to the `save()` and `load()` actions so that the names of the action types & creators match. Personally I'd prefer removing the suffix, unless you plan to add a separate `SAVE` action later?